cli/src/main/java/com/devonfw/tools/ide/tool/eclipse/Eclipse.java

```diff
@@ -80,7 +81,12 @@ protected ProcessResult runEclipse(boolean log, String... args) {
     Path javaPath = getCommandlet(Java.class).getToolBinPath();
     pc.addArg("-vm").addArg(javaPath);
     pc.addArgs(args);
-    return pc.run(log, false);
+
+    if (log) {
+      return pc.run(ProcessMode.BACKGROUND);
+    } else {
+      return pc.run(ProcessMode.BACKGROUND_SILENT);
+    }
```

Comment:
So IMHO we can pass `ProcessMode` as argument to this method instead of `boolean log`.
Then from `installPlugin` we can exec in foreground and from `run` we exec in background.

Reply:
Has been refactored in commit https://github.com/devonfw/IDEasy/pull/200/commits/a1ca068b614e6f2fbae4f97e87bffab762f24981